samples/echoMultiBot/main.go

```diff
@@ -130,7 +130,9 @@ func startWebhookBots(updater *ext.Updater, bots []*gotgbot.Bot, domain string,
 
 	// We add all the bots to the updater.
 	for idx, b := range bots {
-		updater.AddWebhook(b, b.GetToken(), opts)
+		err = updater.AddWebhook(b, b.GetToken(), opts)
+		return fmt.Errorf("failed to add bot: %w", err)
```

Comment:
If my eyes do not betray me, this will terminate on the first iteration with:
"failed to add bot: \<nil\>"
Shouldn't this be 
```suggestion
		if err !=nil { return fmt.Errorf("failed to add bot: %w", err) }
```

?

Reply:
Yes, good catch! Thanks!